components/shared/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# build, lint, and test
+FROM node:12.4 as builder
+WORKDIR /usr/app/builder
+COPY ./components/stream-provider ./components/stream-provider
```

Comment:
i am confused why the shared project has a reference to the stream provider

Reply:
whoops. you should be because its not necessary. removing it